frameworks/PHP/workerman/benchmark_config.json

```diff
@@ -62,7 +104,7 @@
       "webserver": "None",
       "os": "Linux",
       "database_os": "Linux",
-      "display_name": "workerman-postgres",
+      "display_name": "workerman[pgsql]",
```

Comment:
Please use an space or we loss the visibility gain.

workerman[pgsql] vs
workerman [pgsql]